linode/manager
# fix: [M3-8508] - Cancel Button Not Functioning in Delete Node Balancer Configuration Dialog

## Description 📝
This PR fix the issue Cancel Button not functioning inDelete Node Balancer Configuration Dialog.


## Target release date 🗓️
9/30

## Preview

https://github.com/user-attachments/assets/38ebd46f-3cf3-499a-9e17-f6fa5f836aa7


## How to test 🧪

### Verification steps
(How to verify changes)
- Navigate to Nodebalancer, then create a node balancer, then try to delete its configuration and click 'cancel' button in the dialog. 
- Verify that cancel button should function as expected. 
- 
## As an Author I have considered 🤔

*Check all that apply*
